localdashboard_cumulus_std.bash

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Nit: consistency - we're using .sh for most scripts in this and Core - there's no reason we can't use .bash to be more clear in the filename. 

Nit: Please consider *not* leaving scripts in the root of the project.